### DISCUSSION TITLE
19.08.2023

### DISCUSSION THREAD

**Discussion by farag2** [farag2/Sophia-Script-for-Windows#530] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/farag2/Sophia-Script-for-Windows/releases/tag/6.5.5'>19.08.2023</a>.</em>

**Comment by Kolombo98GH** [comment]
Ребята, вопрос – пожалуйста:
**Для Windows 10 Education 2021 выбрать из этого перечня, – именно этот вариант (закрашено жёлтым):**
![GitHub com – Sophia Script for Windows 10 v5 17 5( zip)](https://github.com/farag2/Sophia-Script-for-Windows/assets/139468732/11ec1533-a6d6-4d83-bb3c-7bb738b5830c)
**_— Верно?…_**

_– Здоровенькі були!…_

**Comment by farag2** [comment]
Привет! Я думаю, да. Я не запускал никогда Education, так как нигде ее не видел живьем. Зачем она тебе вообще? :)

**Comment by Kolombo98GH** [comment]
Эт' методическим путём вышел мой выбор, – именно на эту Education-редакцию. – Ну, смотри:

 **•** В Professional много лишнего «Непотребства» …. (И все эти годы неверное было у меня представление, – об этих Pro-редакциях…);
 **•** Тогда, значит, бери и ставь Home-редакцию (в которой почти нет ничего «лишнего», действительно…), **но!** Что у Pro, что у Home (и не только…) нет так называемого _«включения нулевого уровня телеметрии»,_ – одной Политикой (!);
 **•** Такая возможность – _«Включение нулевого уровня телеметрии»_ – есть только у IoT Enterprise 2021 LTSC, Enterprise 2021 LTSC и у Education 2021 – в частности.
**Так вот,** в результате, пал мой выбор, – на Education этот, всё-таки – **т.к.** понял я, что для меня важнее, всё же, эт' относительно частые Обновления, – **дабы,** дескать, идти в ногу со временем (‼), – чего нельзя сказать про эти, как раз таки, LTSC-редакции;

 **• Ну,** а у Windows 11 медленная Оболочка, – как рассказывают, вот, в Сети (!), а у меня Старенький уже системник (P5K SE, – с максимально-возможным апгрейдом (‼)). **Поэтому** – ну её пока, этну 11-ку!…

– Вот так-то, собственно…. B)

**_P.S.:_**
**Фараг2,** раз я у тебя первый, стало быть, эт' с такой, вот, Редакцией – то вот и проверишь на мне тогда, получается, – свою Софию эту на этом Windows 10 Education 2021.
**А потом,** я отпишусь здесь, – расскажу об результате в целом (!?).

**Эм-м,** если у тебя для меня есть, эт' какие-то «примечания» – всенепременно выслушаю тебя, вот….

**М-м-м,** если у меня будут возникать ошибки какие-то, – во время применения София-скрипта, _то сохраниться ли где-то, – по итогу всего процесса, – Лог-файл с этими ошибками?_

– Если что…, Скрипт твой этот планирую настраивать, эт' по средствам **Notepad++.**

**Comment by farag2** [comment]
> В Professional много лишнего «Непотребства»

В Windows нет никакого непотребства.

> нет ничего «лишнего»

Это как, лишнее? в Windows есть что-то лишнее?)

> нет так называемого «включения нулевого уровня телеметрии»

Ну, нет и нет. Всем пофиг.

> Ну, а у Windows 11 медленная Оболочка, – как рассказывают, вот, в Сети

Windows 11 работает быстро. Не надо читать советских газет

> а у меня Старенький уже системник (P5K SE, – с максимально-возможным апгрейдом (‼)).

Как говорил мой преподаватель по матану: "Надо немножко работать". :)

> Эм-м, если у тебя для меня есть, эт' какие-то «примечания» – всенепременно выслушаю тебя, вот….

Так слоэно сказать сразу. Пиши в группу Telegram. Там ответят на все: https://t.me/sophia_chat

> Лог-файл с этими ошибками?

В самом конце отработки всего скрипта будет выведены ошибки, если они будут. Или включить изначально транскрипт в пресете для ведения вообще всего лога. Но скрин сводки по ошибкам в конце мне хватит.

> Скрипт твой этот планирую настраивать, эт' по средствам Notepad++.

Правильный выбор. :)

**Reply by Kolombo98GH** [reply]
**Ну да,** про ваш Соответствующий телеграм приходилось уже слыхать (!) мне….
**Но** у меня с Телеграмом траблы определённые (нет пока возможности у меня, – зарегистрироваться в нём (‼)).
**И по сему, — вопрос, пожалуйста:**

_А есть ли этому самому Телеграму Альтернативное в Сети место у Вас – **ну, например,** Форум какой-то, но по соответствию чтоб, само собой разумеется, **или, например** – Дискорд?…_
(Хотя, думаю, – если б было бы такое Альтернативное место у Вас…, то Ты рассказал бы мне и об этом месте, полагаю).

**_P.P.S.:_**
Про «лог», эт' всё, вроде как, понятно теперь мне. _– Спасибо._

**Reply by farag2** [reply]
https://github.com/farag2/Sophia-Script-for-Windows#media

**Reply by farag2** [reply]
так тяжело найти, да?)

**Reply by farag2** [reply]
> нет пока возможности у меня, – зарегистрироваться

Почему? :D

**Reply by Kolombo98GH** [reply]
_– «Так тяжело найти, да?»_ – риторический вопрос… (!?).
_— Спасибо._

_– «Почему? :D»_ **—** у меня всё ещё BlackBerry, поддержка которого прекращена давным-давно уже: никакое Современное ПО не запустится уже на нём….
**—** Телеграм привязан к номеру (такая у Них политика, и ничего не поделаешь с этим (!?)), что означает – необходимую установку Телеграма на «Телефон» и последующую там регистрацию потом. **И уже потом,** можно осуществлять, – синхронизацию с Телеграмом на ПК, Ноуте и т.д., – **имея, опять-таки,** Номер телефона и Мобильный телеграм на нём.
**—** Есть обход, как бы, – через Гугл-звонок (и через подобные ему Приложения в Сети…). **Но там,** эт' такие танцы с бубном, – капец! **Да и не факт,** – что сработает в результате…. (Интересовался вплотную этой Темой, было дело…).
**—** А Соответствующего эмулятора нет (ну, – как, например, существует такое дело по Вайберу, – эмулятор там какой-то для ПК, Ноута и т.д.). **Ну, по крайней мере,** никаких Эмуляторов по Телеге не было тогда, – в тот период, когда я не на шутку интересовался этой Темой….

**_— Почему же у меня до сих пор BlackBerry?…_** – Возвращаемся и здесь к "твоему" тезису _— «Надо немножко работать»._
**Но!** _«Немножко»_ заменил бы на _«множко»,_ – мягко говоря. **А** от работы-то кони-то до-о-охнут _– а чем же мы хуже этих коней?_ **А** когда Работа является, по сути, – олицетворением «синдрома самозванца» – то, тем более!

я — творческий человек (‼). И в свои сорок, вот, эт' ещё не научился я, эт' монетизировать своё Творчество в Ин-нете (!). – Т.к. 20+ лет играл, как раз таки, в эту игру жизни _— «Синдром самозванца»,_ эт' вместо того, чтоб заниматься по жизни, это в основном тем, чем всегда хотелось заниматься, что любишь… – т.е., своим определённым – и не очень – творчеством (‼).
**И,** пока я учусь зарабатывать, эт' Творчеством своим…, приходится мне, – как тому Художнику (!), ну, давай по-другому выражусь, – приходится быть пока аскетом, типа, и иждивенцем (!).
**Да и** приходится иметь дела, эт' с той Техникой (‼), это которая имеется, вот, собственно…. **—** Хорошо, что хоть такая имеется у меня в распоряжении, эт' и для затянувшегося старта, наконец-то!… ヅ

**Reply by farag2** [reply]
Понятно. :) но разве, если поставить TG на ПК, то телефон и не нужен. Хотя бы так. Там, вроде, придет лишь SMS-код.

**Reply by Kolombo98GH** [reply]
Ну, вот смотри, – (применял только что Портативную версию, 4.9.2.0) так выглядит по стандартному это:
![Telegram exe(4 9 2 0) – Условия синхронизации #01 – Отправить код по SMS](https://github.com/farag2/Sophia-Script-for-Windows/assets/139468732/742ccb6d-320a-4bfb-804f-dcac078d18f9)
![Telegram exe(4 9 2 0) – Условия синхронизации #02 – результат (!)](https://github.com/farag2/Sophia-Script-for-Windows/assets/139468732/806d5342-6838-42f6-9b55-7c83a93c4224)

**Reply by Kolombo98GH** [reply]
**А вот, кстати, и возможное решение…,** это про которое я упоминал здесь выше, вот, но полагал тогда (т.к. тогда не попалось мне это на глаза, почему-то…), – что пока ещё нет, эт' такого Эмулятора, но он есть, оказывается – **вот:**
https://www.youtube.com/watch?v=EH4e_N9comU	< (07:27) **Как зарегистрировать и установить Телеграм на Ноутбук через СМС без установки на Смартфон** – (22.08.2022) ….
**—** По свободе буду пробовать сей Эмулятор, – обязательно: уже давно имеется потребность у меня, – в этом самом Телеграме!…

**Reply by farag2** [reply]
Дерзай!
